extensions/wikia/UserLogin/js/UserLoginFacebook.js

```diff
@@ -142,7 +146,7 @@
 
 			// logged in using FB account, reload the page or callback
 			if (response.loggedIn) {
-				this.loggedInCallback();
```

Comment:
Hm, I think it's okay to have gotten rid of lines 171-187, but `this.loggedInCallback()` was also checking if there was a `loginCallback` set by another extension. I believe the only place that sets this is the CreateNewWiki flow. It's really not a great system, but I haven't had a chance to refactor that aspect yet. In any case, this change might break that feature.